### PR TITLE
Add kube2iam helm chart

### PR DESCRIPTION
**Pull request by NissesSenap**

### CODE REVIEW

File: modules/kubernetes/kube2iam/templates/values.yaml.tpl [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  interface: cali+
+  port: 8181
+
+prometheus:
+  # Port to expose the /metrics endpoint on. If unset, defaults to `host.port`
+  # metricsPort: 9543
+  service:
+    enabled: true
+
+  serviceMonitor:
```
Comment (phillebaba) [inline modules/kubernetes/kube2iam/templates/values.yaml.tpl:23]: Remove and define in prometheus module

File: modules/kubernetes/kube2iam/templates/values.yaml.tpl [hunk 2]
```diff
@@ -0,0 +1,38 @@
+extraArgs:
+  namespace-restrictions: true
+
+#   base-role-arn: arn:aws:iam::0123456789:role/
```
Comment (phillebaba) [inline modules/kubernetes/kube2iam/templates/values.yaml.tpl:4]: no need to have any of these

File: modules/kubernetes/kube2iam/templates/values.yaml.tpl [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    # Labels to add to the service monitor
+    labels:
+      xkf.xenit.io/monitoring: platform
+
+priorityClassName: "platform-high"
+
+rbac:
+  ## If true, create & use RBAC resources
+  ##
+  create: true
```
Comment (phillebaba) [inline modules/kubernetes/kube2iam/templates/values.yaml.tpl:35]: is this the same as the default value? If so it can be removed

**Review by phillebaba — COMMENTED** [review]
Some small comments, but you need to add the module to eks-core.

File: modules/kubernetes/kube2iam/templates/values.yaml.tpl (hunk 1 above)
Comment (NissesSenap) [reply modules/kubernetes/kube2iam/templates/values.yaml.tpl:18]: done

File: modules/kubernetes/kube2iam/templates/values.yaml.tpl (hunk 3 above)
Comment (NissesSenap) [reply modules/kubernetes/kube2iam/templates/values.yaml.tpl:27]: psp is deafult but not rbac.

**Comment by NissesSenap** [comment]
We have decided to not go with kube2iam

**Pull request closed without merging**